### PR TITLE
Add gado to report template

### PR DESCRIPTION
**Pull request by bartcharbon**

### CODE REVIEW

File: src/components/record/info/Hpo.tsx [hunk 1]
```diff
@@ -19,6 +16,9 @@ export const Hpo: Component<FieldProps> = (props) => {
       href: `https://hpo.jax.org/app/browse/term/${encodeURI(hpoTermId)}`,
     }));
 
+  const gadoIndex = getCsqInfoIndex(props.infoMeta, "GADO_PD");
```
Comment (dennishendriksen) [inline src/components/record/info/Hpo.tsx:19]:
```suggestion
  const gadoIndex = (): number => getCsqInfoIndex(props.infoMeta, "GADO_PD");
```

File: src/components/record/info/Hpo.tsx [hunk 2]
```diff
@@ -19,6 +16,9 @@ export const Hpo: Component<FieldProps> = (props) => {
...
+  const gadoClass = gadoIndex != -1 ? (getCsqInfo(props.info, gadoIndex) as string) : null;
```
Comment (dennishendriksen) [inline src/components/record/info/Hpo.tsx:20]:
```suggestion
  const gadoClass = (): string | null => (gadoIndex() !== -1 ? (getCsqInfo(props.info, gadoIndex()) as string) : null);
```

File: src/components/record/info/Hpo.tsx [hunk 3]
```diff
@@ -31,6 +31,22 @@ export const Hpo: Component<FieldProps> = (props) => {
           </>
         )}
       </For>
+      <Show when={gadoClass !== null && gadoClass === "LC"} keyed>
```
Comment (dennishendriksen) [inline src/components/record/info/Hpo.tsx:34]:
```suggestion
      <Show when={gadoClass() !== null && gadoClass() === "LC"} keyed>
```

File: src/components/record/info/Hpo.tsx [hunk 4]
```diff
@@ -1,10 +1,7 @@
 import { Component, createMemo, For, Show } from "solid-js";
```
Comment (dennishendriksen) [inline src/components/record/info/Hpo.tsx:1]:
```suggestion
import { Component, For, Show } from "solid-js";
```

**Review by dennishendriksen — CHANGES_REQUESTED** [review]

File: src/views/SampleVariants.tsx [hunk 5]
```diff
@@ -85,9 +85,14 @@ export const SampleVariants: Component<{
       actions.setSampleVariantsFilterQuery(
         props.sample,
         {
-          selector: selectorHpo,
-          operator: "any_has_any",
-          args: props.samplePhenotypes.map((phenotype) => phenotype.type.id),
+          operator: "or",
```
[the remark this answers is not in the document]
Comment (bartcharbon) [reply src/views/SampleVariants.tsx:113]: it could, be due to other changes now it can't be removed anymore ;)

**Review by dennishendriksen — APPROVED** [review]

**Pull request merged**